NSS-Day-Cohort-55/snackoverflow-potatoes-arent-real
# Fs register and databse

# Description

>Registration functionality added to RegisterForm.js and UserManager.js
>Order buttons only show if logged in, changes in FoodsList.js and FoodCard.js
>Databse.json adjusted to have orders and oderFoods tables.

Fixes # (issue)

## Type of change


# Testing Instructions

1. Register a new account and then log in.
2. Check for successful account creation and logging in.
3. Order buttons should only show when logged in.


# Checklist:
